application/src/main/resources/thingsboard.yml

```diff
@@ -544,6 +544,12 @@ spring:
     hikari:
       maximumPoolSize: "${SPRING_DATASOURCE_MAXIMUM_POOL_SIZE:16}"
 
+# spring configuration for caching static resources (1 year by default)
+spring.mvc.cache:
```

Comment:
Please move this to the existing 'cache' section. The sub-section may be called 'ui' (similar to 'attributes' and 'CACHE_ATTRIBUTES_ENABLED'). The property naming that you suggest is a bit misleading. Developers may think that this is regular Spring Boot configs but this is a completely custom config parameters that are specific to our implementation.